### PR TITLE
MintTo and BurnFrom

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

File: contracts/cw-usdc/src/execute.rs [hunk 1]
```diff
@@ -58,8 +58,12 @@ pub fn mint(
 pub fn burn(
     deps: DepsMut,
     info: MessageInfo,
+    from_address: String,
     amount: Uint128,
 ) -> Result<Response<OsmosisMsg>, ContractError> {
+    // validate that to_address is a valid address
```
Comment (thomasWos) [inline contracts/cw-usdc/src/execute.rs:64]:
```suggestion
    // validate that from_address is a valid address
```

Comment (thomasWos) [reply contracts/cw-usdc/src/execute.rs:64]: Stupid question... and maybe this is just in progress
but why do we pass `from_address` and just validate it?

Comment (sunnya97) [reply contracts/cw-usdc/src/execute.rs:64]: This is still in progress 😄

**Comment by sunnya97** [comment]
@iboss-ptk feel free to close this PR

**Pull request closed without merging**